### ISSUE TITLE
Is this repo still supported/active?

### ISSUE THREAD

**Issue by bhellman1** [lynndylanhurley/redux-auth#119]
I'm noticing the warnings like seen below. Is this repo still being actively maintained / supported?

Warning: Accessing PropTypes via the main React package is deprecated. Use the prop-types package from npm instead.

Warning: You are manually calling a React.PropTypes validation function for the `formControlFeedback` prop on `Glyphicon`. This is deprecated and will throw in the standalone `prop-types` package. You may be seeing this warning due to a third-party PropTypes library. See https://fb.me/react-warning-dont-call-proptypes for details.

**Comment by waleedarshad** [comment]
@bhellman1  is it working for you? what other solution you tried for authentication?

**Comment by betoharres** [comment]
yeah it is for quite some time, it is a great library btw, but it needs a rewrite, or at least a great code review

**Comment by thesubr00t** [comment]
Any news?

**Comment by betoharres** [comment]
@thesubr00t nope, actually I did my own `redux-auth` (client-side only using `redux-form`) and it took me only a week to build. It's not that hard.

**Comment by thesubr00t** [comment]
@betoharres thank you for your reply!

Would you mind to share a gist? It would be helpful.

Thanks,
